udger.go

```diff
@@ -52,66 +37,51 @@ func New(dbPath string, flags *Flags) (*Udger, error) {
 	return u, nil
 }
 
-// Lookup one user agent and return a Info struct who contains all the metadata possible for the UA.
+// Lookup returns all the metadata possible for the given user agent string ua.
 func (udger *Udger) Lookup(ua string) (*Info, error) {
 	info := &Info{}
-	f := udger.Flags
 
-	var browserID int
-	if f.browser {
-		browserID, version, err := udger.findDataWithVersion(ua, udger.rexBrowsers, true)
-		if err != nil {
-			return nil, err
-		}
-		info.Browser = udger.Browsers[browserID]
-		if info.Browser.Family != "" {
-			info.Browser.Name = info.Browser.Family + " " + version
-		}
-		info.Browser.Version = version
-		info.Browser.Type = udger.browserTypes[info.Browser.typ]
+	browserID, browserVersion := udger.findDataWithVersion(ua, udger.rexBrowsers, true)
+	info.Browser = udger.Browsers[browserID]
```

Comment:
We are not handling the case when browserID = -1.

Reply:
Empty browser is used in that case. But I added a unit test to check the lookup with an empty string and found out the returned value was not empty. I fixed it.